test/docs/testplans/conntrack.md

```diff
@@ -0,0 +1,85 @@
+# Table of content
+
+1. [Objectives](#objectives)
+2. [Requirements](#requirements)
+3. [Automation](#automation)
+4. [Test Suites](#test-suites)
+    - [Basic](#eni-creation)
+    - [Ageing](#eni-removal)
+    - [Performance](#eni-scale)
+
+---
+
+# Objectives
+
+Verify proper functioning of the connection tracking mechanism: establishing, handling, closing connections.
+Connection per Second (CPS) is the most important attribute of the DASH products.
+
+# Requirements
+
+| Item |	Expected value
+|---|---
+| Active Connections/ENI | 1M (Bidirectional)
+| CPS per card | 4M+
+
+**To clarify**
+1. Ageing time
+
+# Automation
+
+Test cases are automated:
+1. Functional - using SAI PTF test framework.
+1. Scale/Performance - to be defined.
+
+## Automation notes
+
+1. Each test should be executed using TCP and UDP streams.
+1. Each test should be executed for Inbound and Outbound routing scenario.
+1. So far there is **no direct API to get active connection number** or CPS. Verification might be done using indirect ways:

```

Comment:
This needs to be brought to the Behavioral Model working group. Perhaps we need to agree on some DASH SAI APIs for "telemetry" type usages (possibly fixed headers, i.e. non-P4-derived)